lib/segment/src/payload_storage/in_memory_payload_storage_impl.rs

```diff
@@ -91,6 +91,19 @@ impl PayloadStorage for InMemoryPayloadStorage {
     fn files(&self) -> Vec<PathBuf> {
         vec![]
     }
+
+    fn get_storage_size_bytes(&self) -> OperationResult<usize> {
```

Comment:
I came up with a super simple implementation for the in-memory payload storage as it is only used in tests.
Feel free to suggest a better one if you want :)

Reply:
I was already writing a huge comment about performance until I noticed

> only used in tests